terraform/main.tf

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+resource "juju_application" "mongodb-k8s" {
+
+  charm {
+    name     = "mongodb-k8s"
+    channel  = var.channel
+    revision = var.revision
+    base     = "ubuntu@22.04"
```

Comment:
in the future, this will be extracted in its own variable when we support dual bases